osu.Server.Queues.ScoreStatisticsProcessor/Commands/Performance/PerformanceCommand.cs

```diff
@@ -69,7 +69,7 @@ protected async Task ProcessUserTotals(int[] userIds, CancellationToken cancella
                     if (userStats == null)
                         return;
 
-                    await TotalProcessor.UpdateUserStatsAsync(userStats, RulesetId, db);
+                    await TotalProcessor.UpdateUserStatsAsync(userStats, RulesetId, db, updateIndex: false);
```

Comment:
If this is going to be purposefully skip updating the rank index, I assume that something is _eventually_ going to ensure that the index is correct?

In other words: there is a scheduled process that updates all users' rank index once a day or something, to ensure there are no three rank 1s or something, correct?

Reply:
Yep, you can check `update_pp_rank_history.php` if curious. I generally pause this while doing any batch updates then let it run the next feasible day.